activities/live-streaming/prepare-live-stream.md

```diff
@@ -28,8 +28,8 @@ When setting up a new [live stream](index.md) for 'Let's talk about public code'
 
 Make sure these are ready at least 3 weeks in advance:
 
-- Text messages for social media
-- Images and graphs that can be used in the communication
+- Text messages for social media. Use Planable for the scheduling of those.
```

Comment:
When this ends with a full stop, should all bullet points have full stop? I am unsure. Thoughts @ElenaFdR